dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Machines/VoltageRegulators/Standard/Scrx.mo

```diff
@@ -30,15 +30,17 @@ model Scrx "Bus-fed or solid-fed exciter model, with no excitation resistance"
   Modelica.Blocks.Interfaces.RealInput IrPu(start = Ir0Pu) "Rotor current in pu (base SNom, user-selected base voltage)" annotation(
     Placement(visible = true, transformation(origin = {-180, 120}, extent = {{-20, -20}, {20, 20}}, rotation = 0), iconTransformation(origin = {120, -80}, extent = {{20, -20}, {-20, 20}}, rotation = 0)));
   Modelica.Blocks.Interfaces.RealInput UOelPu(start = 0) "Overexcitation limitation output voltage in pu (base UNom)" annotation(
-    Placement(visible = true, transformation(origin = {-180, 40}, extent = {{-20, -20}, {20, 20}}, rotation = 0), iconTransformation(origin = {-120, 80}, extent = {{-20, -20}, {20, 20}}, rotation = 0)));
+    Placement(visible = true, transformation(origin = {-180, 80}, extent = {{-20, -20}, {20, 20}}, rotation = 0), iconTransformation(origin = {-120, 80}, extent = {{-20, -20}, {20, 20}}, rotation = 0)));
   Modelica.Blocks.Interfaces.RealInput UPssPu(start = 0) "Power system stabilizer output voltage in pu (base UNom)" annotation(
-    Placement(visible = true, transformation(origin = {-180, -40}, extent = {{-20, -20}, {20, 20}}, rotation = 0), iconTransformation(origin = {-120, -80}, extent = {{-20, -20}, {20, 20}}, rotation = 0)));
+    Placement(visible = true, transformation(origin = {-180, 0}, extent = {{-20, -20}, {20, 20}}, rotation = 0), iconTransformation(origin = {-120, -80}, extent = {{-20, -20}, {20, 20}}, rotation = 0)));
+  Modelica.Blocks.Interfaces.RealInput UPu(start = U0Pu) "Stator voltage in pu (base UNom)" annotation(
```

Comment:
Do you mean terminal?

I am confused: the stator voltage is the terminal voltage. Which terminal you are talking about? This is in the documentation?

Reply:
I have replaced the description by : "Voltage amplitude at terminal in pu (base UNom)"

In GeneratorSynchronous.mo, I read 

UPu = ComplexMath.'abs'(terminal.V);
uStatorPu = 1 / rTfoPu * (terminal.V - terminal.i * Complex(RTfoPu, XTfoPu) * SystemBase.SnRef / SNom);
UStatorPu.value = ComplexMath.'abs'(uStatorPu);

Therefore UPu = UsPu <=> RTfoPu = XTfoPu = 0 and rTfoPu = 1

Reply:
In 1024625_Common Information Model _CIM_ for Dynamics_ Standard Models.pdf, the following diagram can be found :
![image](https://github.com/dynawo/dynawo/assets/77326507/f5647ae7-bc2b-40ef-96bf-5ffd4d8306fe)

Reply:
Vc is the filtered stator voltage, Vs the PSS output voltage, Vt the voltage at the terminal.

Reply:
In our synchronous generator model, we added this transformer, but usually, we never use it, so UPu = UsPu all the time.

When we have standard regulations, they always define a generator model without a transformer. The terminal of the generator model is always the stator voltage. 
Unless they define it with words that Vt is the voltage behind the transformer, Vt is the stator voltage.

We can keep it as it is

Reply:
Thanks for the explanation. I am reversing this change.